### ISSUE TITLE
Implement additional evaluation metrics

### ISSUE THREAD

**Issue by fluegelk** [Helmholtz-AI-Energy/special-couscous#16]
Implement additional evaluation metrics for more extensive evaluation of breaking IID experiments.
Each metric receives the confusion matrix (read from csv) of a local or global classifier and returns the metric based on this data.

- [x] Collect list of all metrics to implement (all required information must be available from the confusion matrix)
  - [x] Go through [`sklearn`'s list of classification metrics](https://scikit-learn.org/stable/modules/model_evaluation.html#classification-metrics)
- [x] Check if `pandas` or `sklearn` already provide a way to compute these metrics from a confusion matrix
  - seems like they use sample-wise inputs and convert to confusion matrix internally
- [x] Implement confusion matrix IO: at least reading confusion matrix from csv
  - the confusion matrix is a `numpy.ndarray` so we can use [`numpy.loadtxt`](https://numpy.org/doc/2.1/reference/generated/numpy.loadtxt.html#numpy.loadtxt)/[`savetxt`](https://numpy.org/doc/stable/reference/generated/numpy.savetxt.html) to read/write it
- [x] Implement metrics:
  - [x] [Accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html#sklearn.metrics.accuracy_score)
  - [x] [Balanced Accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.balanced_accuracy_score.html#sklearn.metrics.balanced_accuracy_score)
  - [x] Precision, recall and F-measures with different averaging methods
    - [x] [Precision](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html#sklearn.metrics.precision_score)
    - [x] [Recall](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html#sklearn.metrics.recall_score)
    - [x] [F1](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html#sklearn.metrics.f1_score)
    - [x] [Fβ](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.fbeta_score.html#sklearn.metrics.fbeta_score)
    - [x] All averaging methods from sklearn: "micro", "samples", "macro", "weighted", None, see definition [here](https://scikit-learn.org/stable/modules/model_evaluation.html#multiclass-and-multilabel-classification)
  - [x] [Cohen's Kappa](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.cohen_kappa_score.html#sklearn.metrics.cohen_kappa_score)
  - [x] [Matthews correlation coefficient (MCC)](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html#sklearn.metrics.matthews_corrcoef)
- [x] Write tests
  - consider using [`hypothesis`](https://hypothesis.readthedocs.io/en/latest/quickstart.html) and comparing against `sklearn`'s implementation from sample-wise inputs → moved to a new issue, only manual tests for now
- [x] Add short description of all introduced metrics, e.g. to readme
